### PR TITLE
Add SL1K2 flow meter

### PR DESCRIPTION
**Pull request by nrwslac**
## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive comments
- [ ] Test suite passes locally
- [ ] Libraries are set to fixed versions and not ``Always Newest``
- [ ] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

File: lcls-plc-rixs-optics/_Config/PLC/rixs_optics.xti [hunk 1]
```diff
@@ -4337,8 +4345,8 @@ Emergency Stop for MR1K1]]></Comment>
 				<Link VarA="PlcTask Inputs^Main.M23.nRawEncoderULINT" VarB="FB Inputs Channel 1^Position" AutoLink="true"/>
 			</OwnerB>
 			<OwnerB Name="TIID^Device 1 (EtherCAT)^Term 1 (EK1200)^E5 (EK1122)^SL1K2-EXIT (EK1100)^EL7047_SL1K2_GAP">
-				<Link VarA="PlcTask Inputs^Main.M22.bLimitBackwardEnable" VarB="STM Status^Status^Digital input 1" AutoLink="true" Size="1"/>
-				<Link VarA="PlcTask Inputs^Main.M22.bLimitForwardEnable" VarB="STM Status^Status^Digital input 2" AutoLink="true" Size="1"/>
+				<Link VarA="PlcTask Inputs^Main.M22.bLimitBackwardEnable" VarB="STM Status^Status^Digital input 2" AutoLink="true" Size="1"/>
```
Comment (nrwslac) [inline lcls-plc-rixs-optics/_Config/PLC/rixs_optics.xti:4348]: Not sure why this changed? I didn't modify these. Is bLimitBackwardEnable supposed to be linked to input 1? @ghalym

Comment (nrwslac) [reply lcls-plc-rixs-optics/_Config/PLC/rixs_optics.xti:4348]: bLimitBackwardEnable on 2 as it is here is consist with the previous pull request, but master has bLimitBackwardEnable on 1.

Comment (ghalym) [reply lcls-plc-rixs-optics/_Config/PLC/rixs_optics.xti:4348]: Interesting. Checking my notes the forwards is input 1 and the backwards is input 2. These changes should have been committed. I also changes the limits for other axes. I will take another look.

**Review by ghalym — APPROVED** [review]
lgtm

**Pull request merged**